addons/ofxOsc/addon_config.mk

```diff
@@ -78,3 +78,7 @@ vs:
 	# a specific platform
 	ADDON_SOURCES_EXCLUDE = libs/oscpack/src/ip/posix/%
 	ADDON_DEFINES = OSC_HOST_LITTLE_ENDIAN
+	
+osx:
+	ADDON_INCLUDES_EXCLUDE = libs/oscpack/src/ip/win32/%
+	
```

Comment:
@dimitre - I think we would need this for all other platforms too.

Reply:
Great, I'll change the PR to a common block

Reply:
done